.github/workflows/release.Hutch.Relay.yml

```diff
@@ -0,0 +1,48 @@
+name: Release Hutch.Relay to Docker Hub
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - app/Hutch.Relay/**
+      - test/Hutch.Relay.Tests/**
+      - .github/workflows/release.Hutch.Relay.yml
+
+env:
+  namespace: hutchstack
+  repo-name: hutch-relay
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v4
+
+      - name: Docker Login
+        uses: docker/login-action@v3.1.0
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
We'll be pushing images to the Github Container registry, not Dockerhub. So best to rename these accordingly, for example `GITHUB_TOKEN`.

Also then set the `registry` as `ghcr.io` - as I think it will default to Dockerhub also.